Tools/CISupport/ews-build/config.json

```diff
@@ -310,7 +338,13 @@
       "name": "JSC-Tests-arm64-EWS", "shortname": "jsc-arm64", "icon": "buildAndTest",
       "factory": "JSCBuildAndTestsFactory", "platform": "mac-monterey",
       "configuration": "release", "runTests": "true",
-      "workernames": ["ews136", "ews137"]
+      "workernames": ["ews137"]
+    },
+    {
+      "name": "JSC-Tests-Ventura-arm64-EWS", "shortname": "jsc-ventura-arm64", "icon": "buildAndTest",
```

Comment:
Since JSC queue is independent from other mac queues, it's possible to migrate this queue independently. Without the need for this long process of adding new queue, migrating machines and deleting old queue. Can just have a small PR to change the platform for `JSC-Tests-arm64-EWS` and update/move the two corresponding bots.